packages/contact-exchange/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@datev/contact-exchange",
```

Comment:
Remove the reference to datev please. In this libraries we don't want references to datev in the codebase.